metrics/time/launch_times.sh

```diff
@@ -38,6 +38,31 @@ REQUIRED_CMDS=("bc" "awk")
 # Note the no-0-padding - 0 padding the results breaks bc in some cases
 DATECMD="date -u +%-s:%-N"
 
+# This the minimum entropy level produced
+# by haveged is 1000 see https://wiki.archlinux.org/index.php/Haveged
+# Less than 1000 could potentially slow down cryptographic
+# applications see https://www.suse.com/support/kb/doc/?id=7011351
+entropy_level="1000"
+
+check_entropy_package() {
+	entropy_package="haveged"
+	# Check that entropy package is installed
+	check_package=$(which ${entropy_package} | wc -l)
+	if [ ${check_package} -eq 0 ]; then
+		die "No entropy package installed"
+	fi
+	# Check that service is running
+	sudo service ${entropy_package} status | grep "running"
+}
+
+measure_entropy() {
```

Comment:
I think that `check_entropy_level` is a better name for this function, what do you think ?